xml/System.Security.Cryptography.X509Certificates/X509ChainPolicy.xml

```diff
@@ -432,17 +432,19 @@
         <ReturnType>System.DateTime</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>The time that the certificate was verified expressed in local time.</summary>
+        <summary>Gets or sets the time for which the chain is to be validated, expressed in local time.</summary>
         <value>A <see cref="T:System.DateTime" /> object.</value>
         <remarks>
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- This property returns a value indicating the time the certificate was validated, expressed in local time. This is important when validating signed messages, since the signature must have been valid at the time of signing, not at the time of validation.  
-  
-   
-  
-## Examples  
+This property returns a value indicating the time the chain is to be validated, expressed in local time. This is important when validating signed messages, since the signature must have been valid at the time of signing, not at the time of validation.
```

Comment:
The "expressed in local time" is both correct and incorrect.  If the DateTimeKind is Local, it's local.  If the DateTimeKind is Utc it's UTC.  If the DateTimeKind is Unknown it is assumed to be Local.  I don't know how we normally express that.

Reply:
Should we just remove that then @bartonjs? @rpetrusha any suggestions here?

Reply:
@mariaw and @bartonjs, I think I would say "expressed either as local time or UTC, depending on the value of the \<xref:System.DateTime.DateTimeKind?displayProperty=nameWithType> property"